GWAS/PCA.ipynb

```diff
@@ -458,7 +553,7 @@
     "# Here genoFile is samples to be projected back\n",
     "parameter: pca_model = path\n",
     "input: genoFile, phenoFile, pca_model\n",
-    "output: f'{_input[1]:n}.projected.rds'\n",
+    "output: f'{_input[1]:n}.{name}.projected.rds'\n",
```

Comment:
Let's output to `cwd` and not use input folder.

Reply:
For inputs derived from previous output, eg in `project_sample_2` , `flash_pca_2` case there is no need to specify output to `cwd` because the input are in cwd already. For when input is specified, let's output them to `cwd` in case input original data folder is messed up.